src/routes/router.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const {getCandies,addNewCandy, getCat, getFlav,deleteCandies,getAddForm,oneFlav, oneCat} = require("../controller");
+
+const router = express.Router();
+
+router.get("/candies", getCandies);
+router.post("/add_candy", addNewCandy);
+router.delete("/delete/:id",deleteCandies);
+router.get("/categories",getCat);
+router.get("/flavors",getFlav);
+router.get("/addCandyForm",getAddForm);
+router.get("/oneFlav/:flavor",oneFlav);
+router.get("/oneCat/:category",oneCat);
+
```

Comment:
You can refactor the code here to make it more readable by separating the routes.

For example, you can create two files to handle `candies` and `flavours`

check the files from the repo I created with you (Night session) 👉 [db-week Repo](https://github.com/karam-zomlut/db-week)